reconcile/utils/ocm.py

```diff
@@ -578,6 +638,24 @@ def __init__(
             self.get_aws_infrastructure_access_role_grants
         )
 
+        # organization sectors and their dependencies
+        self.sectors: dict[str, Sector] = {}
+        for sector in sectors or []:
+            deps = []
+            for dep in sector.get("dependencies") or []:
+                dep_ocm_org_name = self.name
+                if dep.get("ocm"):
+                    dep_ocm_org_name = dep["ocm"]["name"]
+                ref = SectorWeakReference(
+                    ocm_org_name=dep_ocm_org_name, sector_name=dep["name"]
+                )
+                deps.append(ref)
+            self.sectors[sector["name"]] = Sector(
+                name=sector["name"],
+                ocm=self,
+                dependencies_refs=deps,
+            )
+
```

Comment:
Should we split our OCM definition into OCM environments and OCM organizations or accounts? Right now we don't use the concept of an organization in our schema, it is just bound to the account (user/service account) we use to authenticate with the OCM CS. 
If this is expected to grow maybe it's better to have a clear schema definition to handle multiple OCM organizations in clear path forward instead of using the current OCM schema definition.

Reply:
We could, but I am not sure what it would bring.
Our ocm schema is actually and ocm org. Its URL refers to the env.
I think it's out of scope of this PR though

Reply:
IMO it adds clarity, our current OCM is an ENV and an Account. The ORG is set into the account. I'm not sure if an account can have multiple organizations or not, but I would rather prefer not to set the ENV URL in all the  "organization" definitions.

I agree that maybe this is out of the scope of this task. I just pointed it out because this snippet has confused me with the semantics used.

Reply:
I agree this can be confusing. I'll resolve this thread as out of scope, but we can certainly have this conversation in teamchat